diff -Naurp dvd+rw-tools-7.0/growisofs.c dvd+rw-tools-7.0.oden/growisofs.c
--- dvd+rw-tools-7.0/growisofs.c	2008-03-05 12:13:35.000000000 +0100
+++ dvd+rw-tools-7.0.oden/growisofs.c	2008-03-05 12:12:17.000000000 +0100
@@ -419,6 +419,7 @@
 #include <sys/stat.h>
 #include <assert.h>
 #include "mp.h"
+#include <limits.h>
 
 #if defined(__unix) || defined(__unix__)
 # include <unistd.h>
diff -Naurp dvd+rw-tools-7.0/transport.hxx dvd+rw-tools-7.0.oden/transport.hxx
--- dvd+rw-tools-7.0/transport.hxx	2008-03-05 12:13:35.000000000 +0100
+++ dvd+rw-tools-7.0.oden/transport.hxx	2008-03-05 12:12:48.000000000 +0100
@@ -16,6 +16,7 @@
 #include <fcntl.h>
 #include <poll.h>
 #include <sys/time.h>
+#include <limits.h>
 
 inline long getmsecs()
 { struct timeval tv;
