--- dvd/Makefile.m4
+++ dvd/Makefile.m4	2006/01/30 12:18:00
@@ -166,10 +166,10 @@
 # Linux section
 #
 CC	=gcc
-CFLAGS	+=$(WARN) -O2 -D_REENTRANT
+CFLAGS	+=$(WARN) $(RPM_OPT_FLAGS) -D_REENTRANT
 CXX	=g++
-CXXFLAGS+=$(WARN) -O2 -fno-exceptions -D_REENTRANT
-LDLIBS	=-lpthread
+CXXFLAGS+=$(WARN) $(RPM_OPT_FLAGS) -fno-exceptions -D_REENTRANT
+LDLIBS	=-lpthread
 LINK.o	=$(LINK.cc)
 
 prefix?=/usr/local
